### DISCUSSION TITLE
Feed Maker inside the app?

### DISCUSSION THREAD

**Discussion by omrali0505** [mozzius/graysky#81] [Feature requests]
Feed Maker inside the app

**Comment by mozzius** [comment]
A little outside the scope of the app currently but it would be super cool to see. In the meantime, you can use skyfeed.app

**Reply by greglarsen** [reply]
A simple list feature would be really great. Feeds are more complex than I need.

**Reply by mozzius** [reply]
Lists are for sure coming, they're in the API - deck.blue has them
